docs/sources/mimir/configure/about-versioning.md

```diff
@@ -189,3 +189,5 @@ The following features or configuration parameters are currently deprecated and
   - `-ingester.return-only-grpc-errors`
 - Ingester client
   - `-ingester.client.report-grpc-codes-in-instrumentation-label-enabled`
+- Mimirtool
+  - the flag `--rule-files`
```

Comment:
Nit, I think all our flags are single-dashed.

```suggestion
  - the flag `-rule-files`
```

Reply:
That's not what I see from local testing:

```
➜ mimir (armandgrillet/remove-rule-files-flag) ✗ go run ./cmd/mimirtool/main.go rules lint -rule-files=/Users/armand/Downloads/test.yml
main: error: unknown short flag '-r', try --help
exit status 1
➜ mimir (armandgrillet/remove-rule-files-flag) ✗ go run ./cmd/mimirtool/main.go rules lint --rule-files=/Users/armand/Downloads/test.yml
WARN[0000] flag --rule-files is deprecated, use the argument instead
INFO[0000] SUCCESS: 1 rules found, 0 linted expressions
```

Reply:
Oh I see, you're right. This isn't using golang's standard flag parsing, but `kingpin` instead, that's why it's double dashed.
Thank you for checking.